stdlib/public/runtime/ImageInspection.h

```diff
@@ -106,6 +108,47 @@ void addImageAccessibleFunctionsBlockCallbackUnsafe(const void *baseAddress,
 
 int lookupSymbol(const void *address, SymbolInfo *info);
 
+#if defined(_WIN32)
+/// Configure the environment to allow calling into the Debug Help library.
+///
+/// \param body A function to invoke. This function attempts to first initialize
+///   the Debug Help library. The result of that operation is passed to this
+///   function.
+///
+/// On Windows, the Debug Help library (DbgHelp.lib) is not thread-safe. All
+/// calls into it from the Swift runtime and stdlib should route through this
+/// function.
+SWIFT_RUNTIME_STDLIB_SPI
+void _swift_withWin32DbgHelpLibrary(
```

Comment:
Is this really meant to be exposed?

Reply:
Yes, see my comment earlier in the PR:

> This function should be exported so that it can be used by other Swift libraries in the repo if needed. It's not sufficient for those libraries to do their own locking; they would need to use the same lock to ensure thread safety.

I do intend to use it in `_ExceptionHandling` (née `_Cxx`.)